gsoc/background.md

```diff
@@ -1,53 +1,55 @@
+more OpenAstronomy project ideas. Each summer, [several projects][oa projects] are offered by
 [OpenAstronomy mentors][oa mentors], with well-defined scope and pre-requisite
 experience. If you are selected by OpenAstronomy, you will be a GSoC student.
 
 ### What is OpenAstronomy?
 
 OpenAstronomy is an umbrella organization which acts as an interface between
 the Google Summer of Code and several open source, open development projects
-in the astronomical community, including [astropy], [sunpy], and
-[several other projects in the astronomical community][oa members].
-It coordinates mentors and projects for Google Summer of Code mentorships 
-related to astronomy.
+in the astronomical community, including [astropy], [sunpy], and [several
+other astronomy projects][oa members].  It coordinates mentors and projects
+for Google Summer of Code mentorships related to astronomy.
 
 ### What is the application process?
 
 You must submit your application to the [Google Summer of Code][GSoC], but
-it's preferable that you share it before hand with your potential project mentor
-or the rest of the community. They will give you feedback before the deadline.
-Check the [application template][application] or look up for previous years
-applications on the projects' wikis.
+it's preferable that you also share it well before the deadline with your
+potential project mentors or the rest of the community. Then they will be
+able to give you feedback before officially submitting it.  Check the
+[application template][application]. You can also find applications from
+previous years in the GitHub wiki pages of some of the OpenAstronomy
+members.
 
 The complete details on applying are in the [GSoC Student Guide].
 
 ### Who is eligible to apply to GSoC?
 
-OpenAstronomy is actively seeking applicants who are undergraduates or graduate 
-students with experience developing software for astronomy in 
-Python and other languages. We are looking for students who have or are pursuing
-degrees in astronomy (or physics), who are interested in developing open source 
-software. You don't need to be a Python guru or an astronomical coordinate 
-system expert to be eligible -- projects are available that cover a range of 
-previous experience in Python and/or astronomy. Underrepresented students from
-anywhere around the world are especially encouraged to apply.
+OpenAstronomy is actively seeking applicants who are undergraduate or
+graduate students with experience developing software for astronomy in
+Python and other languages. We are especially (but not exclusively) looking
+for students who have or are pursuing degrees in astronomy or physics, more
+importantly who are interested in developing open source software. You don't
+need to be a Python guru or an astronomical coordinate system expert to be
+eligible -- projects are available that cover a range of previous experience
+in Python and astronomy. Underrepresented students from anywhere around
```

Comment:
Is the word `underrepresented` necessary? What does it even mean in the around the world content? The bottom line is really that anyone is really welcome to apply and has a real chance to get selected.

Reply:
I would prefer to emphasize something like ``Any student from any background is very welcome to apply`` instead.

Reply:
Yes, I would prefer that one, too. What do you think @dpshelio?

Reply:
"Students from any background from anywhere around the world is very welcome to apply." ?

Reply:
I agree with @mirca :)